### ISSUE TITLE
[BUG] Unable to Start Song in Multiplayer Lobby

### ISSUE THREAD

**Issue by AlphaTheYveltal** [Goobwabber/MultiplayerExtensions#159]
**Multiplayer Extensions Version and Download Source**
v0.6.2 From Github

**Your Platform**
PC - Oculus Rift S

**Describe the bug**
Whenever my group tries to begin the song, the host cannot begin the song due to the song 'not being owned' or 'no song selected' despite owning/downloading the custom song. When swapped to vanilla songs, the lobby still will not run due to the vanilla songs not being 'owned'.

**To Reproduce**
Owner can create the lobby - which then selecting a custom song will not run at all.

**Expected behavior**
The start button is greyed out.

**Log**
[INFO @ 14:01:09 | MultiplayerExtensions] Entitlement from 'FrJXgCxl9wM1krS6q3B0JB' for 'custom_level_125B07EBCC06FE9667E83FC2D6B9AE5ECBC72E8C' is NotDownloaded

**Screenshots/Video**

**Additional context**
Steam into PC and PC into Steam for the custom songs.

**Comment by TheInevitableLuke** [comment]
I'm curious why this was closed without response - this bug is 100% still happening for me. Only thing I can add is that it also happens on PC with Valve Index, and despite having SiraUtil 3.0.2 installed, I found this in the log:
`[WARNING @ 17:23:05 | IPA/Loader] 'MultiplayerExtensions' is missing dependency 'SiraUtil@^2.5.2'; ignoring`

**Comment by AlphaTheYveltal** [comment]
> I'm curious why this was closed without response - this bug is 100% still happening for me. Only thing I can add is that it also happens on PC with Valve Index, and despite having SiraUtil 3.0.2 installed

Because I found out that in the newest version of the mod, it is required to have BeatTogether installed - so I was just being a dumbass. After installing BeatTogether, it worked after that. Even though it says to have it for over 10+ people, you genuinely need it for the newest version. Don't know why they don't clarify it on the main page and not the install page, but that's how I fixed mine at least.
